test/core/end2end/fixtures/h2_uchannel.c

```diff
@@ -285,7 +285,7 @@ static void chttp2_init_client_micro_fullstack(grpc_end2end_test_fixture *f,
                                                grpc_channel_args *client_args) {
   micro_fullstack_fixture_data *ffd = f->fixture_data;
   grpc_connectivity_state conn_state;
-  grpc_connected_subchannel *connected;
+  grpc_connected_subchannel *is_connected;
```

Comment:
Ah, nice catch. This needed to be renamed from "connected" to some other name to fix a -Wshadow warning treated as error (there's some odd behavior under gcc4.4).